src/nuxt-module.ts

```diff
@@ -13,6 +13,8 @@ const defaultOptions: PlausibleModuleOptions = {
 }
 
 const PlausibleModule: Module<PlausibleModuleOptions> = function (moduleOptions) {
+  if (!this) return;
```

Comment:
Was receiving error by using Vue 3 because "this" didn't existed here. I didn't even used Nuxt here.